src/plugins/rename/rename.c

```diff
@@ -17,33 +17,121 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <stdbool.h>
+#include <stdio.h>
+#include <ctype.h>
 
 #define ELEKTRA_ORIGINAL_NAME_META "origname"
-
+#define TOLOWER (-1)
+#define TOUPPER 1
 // TODO defined privately in keyhelpers.c, API break possible..
 char *keyNameGetOneLevel(const char *name, size_t *size);
 
-Key *elektraKeyCutNamePart(const Key *key, const Key *parentKey, const char *cutPath)
+static void doConversion(char *newName, int levels, int toCase)
```

Comment:
There are two new coding style rules: space after method names and spaces around `*`.
So the line would be:
`static void doConversion (char * newName, int levels, int toCase)`